pages/_app.js

```diff
@@ -1,14 +1,3 @@
-import GlobalStyle from "@/styles";
-import Head from "next/head";
-
-export default function App({ Component, pageProps }) {
-  return (
-    <>
-      <GlobalStyle />
-      <Head>
-        <title>Capstone Project</title>
```

Comment:
you should not remove the Head because it contains the title of your page ..remove capstone project and add your pages title . and even your vercel deploy is not working maybe because of that you should try this out . i read about it and it could happen that the missing head leads to not working app